### ISSUE TITLE
Mobile?

### ISSUE THREAD

**Issue by cgsilvers** [moloch--/RootTheBox#74]
Make interface mobile friendly (e.g. menu items not visible on phone)

**Comment by moloch--** [comment]
I'm hoping to find some time to update to Bootstrap v3 which should help with this, but I'm going to put this under the more long-term goals.

**Comment by cgsilvers** [comment]
Any progress on this? It is the feature that is keeping me from using this scoring engine for the OSINT CTF that we run at several security conferences. information here: https://www.cgsilvers.com/osint-ctf

**Comment by jeffg2k** [comment]
Sorry, haven't updated the bootstrap yet.  The newer versions are not backward compatible so it would take a bit of work.  The menu should be visible if you hold it in landscape on most smartphones, but yeah, it doesn't yet dynamically change the menu structure when in a portrait mode.

**Comment by jeffg2k** [comment]
Maybe we can just tweak the menu to support a responsive format without going through the pain of updating the bootstrap.
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_responsive_navbar_dropdown

**Comment by cgsilvers** [comment]
based on visiting the example using my iPhone, it looks like it would work.

**Comment by jeffg2k** [comment]
@cgsilvers I pushed a change to try and give a menu on the mobile.  It's not pretty, but hopefully it will work to provide some navigation.

**Comment by V0lk3n** [comment]
@eljeffeg sorry i didnt see this issue before open mine. Just want to add something about it. You said to me that generally a computer is needed, but as example, i have nethunter on my phone, and do some CTF / HTB on it. I agree that is not the best, but sometimes if you are borred and outside you can get some little fun :) Specially if its little challenges such as steganography etc. 

By the way, thanks so much to you guys, specially @moloch-- for this awesome project, i was using CTFd and got a lot of problem... take a while to config, well it wasnt really cool. But with this project, it literally took me just one day to setup the whole things with dynamic dns / SSL cert. So yeah, thanks so much to make my life easier :)

**Comment by eljeffeg** [comment]
Yeah, no doubt there are challenges for mobile and would love to get it done.  It's just going to be a time consuming task and I have little experience in migrating Bootstrap versions.

**Comment by V0lk3n** [comment]
> Yeah, no doubt there are challenges for mobile and would love to get it done.  It's just going to be a time consuming task and I have little experience in migrating Bootstrap versions.   If it involves my time, I have to question if the juice is worth the squeeze.

I think i've found something that can work to migrate to bootstrap 3 easily (if it work), and then it should be easy to migrate to 4-5 ^^

https://code.divshot.com/bootstrap3_upgrader/
